layouts/shortcodes/googleVersion.html

```diff
@@ -0,0 +1 @@
+{{- $.Page.Site.Params.softwareVersions.google_cloud_image_version -}}
```

Comment:
I think what you want is just take the `kairos_version` and transform it to your liking here with go instead of having your own variable

Reply:
```suggestion
{{- $.Page.Site.Params.softwareVersions.kairos_version | replace "." "-" -}}
```

Reply:
this will use `master` when user is browsing the `dev` branch no? We don't push master images to google cloud :(

Reply:
@jimmykarily indeed but imo that is ok because at the end of the day, no one should be viewing dev branch unless they know what they are doing in which sense they'd know we don't publish those

Reply:
I guess I coudl also add some functionality so we add an automatic alert on top of those sources that we don't publish on master. I'll create a ticket

Reply:
https://github.com/kairos-io/kairos/issues/3234

Reply:
switching to an older version of the docs will point to older image which we may have cleaned up by that point (we only keep 5 versions around). We don't show very old versions in docs though so this might not be a problem either (or is it?).